lib/events/events.proto

```diff
@@ -1097,6 +1097,73 @@ message AppSessionRequest {
     string Method = 5 [ (gogoproto.jsontag) = "method" ];
 }
 
+// DatabaseMetadata contains common database information.
+message DatabaseMetadata {
+    // DatabaseService is the name of the database service proxying the database.
+    string DatabaseService = 1 [ (gogoproto.jsontag) = "db_service" ];
+    // DatabaseProtocol is the database type, e.g. postgres or mysql.
+    string DatabaseProtocol = 2 [ (gogoproto.jsontag) = "db_protocol" ];
+    // URI is the database URI to connect to.
+    string DatabaseURI = 3 [ (gogoproto.jsontag) = "db_uri" ];
+    // DatabaseName is the name of the database a user is connecting to.
+    string DatabaseName = 4 [ (gogoproto.jsontag) = "db_name" ];
+    // DatabaseUser is the database username used to connect.
+    string DatabaseUser = 5 [ (gogoproto.jsontag) = "db_user" ];
+}
+
+// DatabaseSessionStart is emitted when a user connects to a database.
+message DatabaseSessionStart {
+    // Metadata is a common event metadata.
+    Metadata Metadata = 1
+        [ (gogoproto.nullable) = false, (gogoproto.embed) = true, (gogoproto.jsontag) = "" ];
```

Comment:
I'm not familiar with gogo custom attributes - what would happen if you omitted the empty jsontag?

Reply:
No idea to be honest, I just made it consistent with how other events are defined here.